x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/components/missing_cloud_tier_callout.tsx

```diff
@@ -9,20 +9,23 @@ import { i18n } from '@kbn/i18n';
 import React, { FunctionComponent } from 'react';
 import { EuiCallOut, EuiLink } from '@elastic/eui';
 
-const i18nTexts = {
-  title: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.cloudMissingColdTierCallout.title', {
-    defaultMessage: 'Create a cold tier',
+const geti18nTexts = (tier: 'cold' | 'frozen') => ({
+  title: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.cloudMissingTierCallout.title', {
+    defaultMessage: 'Create a {tier} tier',
+    values: { tier },
```

Comment:
Would you mind confirming with the localization team that this will work for translations? I'm concerned that `tier` variable will always be hard coded to 'cold' or 'frozen' without translation.

Reply:
I think this is a good question. We should keep in mind that `hot`, `warm`, etc are values that have been formalized by the API, which won't be localized. So it might make sense to present them in the UI the same way. I'd like to hear input from people on the i18n team and also from a docs writer on this.

Reply:
I think we should localize the tier name variable (`hot`, warm`, `cold`, `frozen`) in the UI. I assume it would also be translated when we refer to the tier name in other help text. It would seem odd to have an untranslated portion of the string.